parm/config/gfs/config.atmensanl

```diff
@@ -5,9 +5,8 @@
 
 echo "BEGIN: config.atmensanl"
 
-export OBS_YAML_DIR=${HOMEgfs}/sorc/gdas.cd/parm/atm/obs/config/
-export OBS_LIST=${HOMEgfs}/sorc/gdas.cd/parm/atm/obs/lists/lgetkf_prototype.yaml
-export ATMENSYAML=${HOMEgfs}/sorc/gdas.cd/parm/atm/lgetkf/lgetkf.yaml
+export OBS_LIST="${HOMEgfs}/parm/gdas/atm/obs/lists/lgetkf_prototype.yaml.j2"
+export ATMENSYAML="${HOMEgfs}/parm/gdas/atm/lgetkf/lgetkf.yaml.j2"
```

Comment:
```suggestion
export JEDIYAML="${HOMEgfs}/parm/gdas/atm/lgetkf/lgetkf.yaml.j2"
```
Likewise here, if we could rename `ATMENSYAML` to `JEDIYAML`, that would go a long way in yaml rendering.

Reply:
Change committed at ed042392.